src/freenet/clients/http/staticfiles/js/clipboard.js

```diff
@@ -0,0 +1,19 @@
+// @license magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&dn=gpl-3.0.txt GPL-v3-or-Later
+function addCopyKeyFieldToFinishedTransfers() {
+  if (navigator && navigator.clipboard) {
+    var matching = document.getElementsByClassName("finished_key_link");
+    for (var matchingElement of matching) {
+      var key = matchingElement.dataset.key;
+      if (key) {
+        var toClipboard = document.createElement("span");
+        toClipboard.textContent = " ⎘";
```

Comment:
As for the character (⎘), this is not something that I would immediately recognize.

The emoji glyph for clipboard (📋) might be more visually recognizable (depending on OS support). What's your take?

Reply:
The character ⎘ is the only one that’s universally supported. I don’t actually see 📋 on my system (copied it as empty block).

Reply:
Mmh. In that case we might be better off rendering an SVG, as the character may be too font-dependent.

That could be something like this (quick sketch based on distant memories of similar functionality):
```svg
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" style="height: 1em; width: 1em; margin-inline-start: .5ex; vertical-align: middle;">
  <path stroke="currentColor" stroke-width="2" d="M 2 16 v -12 a 2 2 0 0 1 2 -2 h 12 M 9 7 h 10 a 2 2 0 0 1 2 2 v 10 a 2 2 0 0 1 -2 2 h -10 a 2 2 0 0 1 -2 -2 v -10 a 2 2 0 0 1 2 -2" fill="none"></path>
</svg>
```

<img width="183" height="34" alt="afbeelding" src="https://github.com/user-attachments/assets/c929409a-24db-47a2-b1c5-32849c5c3c1a" />